data-tck/src/main/groovy/io/micronaut/data/tck/tests/AbstractTransactionSpec.groovy

```diff
@@ -63,6 +63,10 @@ abstract class AbstractTransactionSpec extends Specification implements TestProp
         return false
     }
 
+    boolean failsInsertInReadOnlyTx() {
```

Comment:
Some drivers don't fail when saving in read only transaction so needed to adjust existing read only tests